dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -613,6 +613,29 @@ def remove_docker_networks(networks: list[str] | None = None) -> None:
             )
 
 
+def remove_docker_volumes(volumes: list[str] | None = None) -> None:
+    """
+    Removes specified docker volumes. If no volumes are specified, it removes all unused volumes.
+    Errors are ignored (not even printed in the output), so you can safely call it without checking
+    if the volumes exist.
+
+    :param volumes: list of volumes to remove
+    """
+    if volumes is None:
+        run_command(
+            ["docker", "volume", "prune", "-f", "-a", "--filter", "label=com.docker.compose.project=breeze"],
```

Comment:
@potiuk , should we do a similar thing when removing networks as well? Since a network created by breeze also has the label "com.docker.compose.project=breeze"

Reply:
Yep. Why not.